api_errors.go

```diff
@@ -0,0 +1,44 @@
+package uadmin
+
+import (
+	"fmt"
+	"net/http"
+)
+
+const (
+	contentTypeHeader = "Content-Type"
+	jsonContentType   = "application/json; charset=utf-8"
+	errStatus         = "error"
+	defaultErrorMsg   = "unknown server error"
+)
+
+var codeToGenericMessage = map[int]string{
```

Comment:
Do we really need this? We can use `http.StatusText(http.Status<code>)`

E.g.,
```
if errMsg == "" {
	errMsg == fmt.Sprintf("%d. %s", code, http.StatusText(code))
}
```